### ISSUE TITLE
turf-erase v. tuf-difference

### ISSUE THREAD

**Issue by sckott** [Turfjs/turf#372]
Seems that this line https://github.com/Turfjs/turf/blob/v3.0.1/packages/turf/index.js#L14 points to `turf-erase`, but I think it's been replaced with https://github.com/Turfjs/turf/tree/master/packages/turf-difference

leads to 

```js
> require ('turf');
Error: Cannot find module 'turf-erase'
```

**Comment by roblav96** [comment]
Anyone know how to fix this? Running nodejs@latest with turf@3.0.1

**Comment by stepankuzmin** [comment]
@roblav96 I've made a PR with fix suggested by @sckott, but unfortunately, I'm getting error saying that there is no `turf-collect` module (although it is in repo and I can load it):

```
Error: Cannot find module 'turf-collect'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/stepan/Documents/projects/turf/packages/turf/index.js:16:14)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

**Comment by tmcw** [comment]
Thanks! Published as turf 3.0.2.
